### PR TITLE
i18n(es): Update `astro-components`, `astro-pages` & `framework-components`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Update `astro-components`, `astro-pages` & `framework-components` based on this commit e788cb13c48ac5b08556123f1cffb603d2d8e1ca

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/core-concepts/astro-components.mdx [hunk 1]
```diff
@@ -378,4 +378,6 @@ Un [elemento `<slot/>`](/es/core-concepts/astro-components/#slots) dentro de un
 
 ## Próximos Pasos
 
-📚 Aprende sobre el uso de [componentes de framework de UI](/es/core-concepts/framework-components/) en tu proyecto Astro.
+import ReadMore from '~/components/ReadMore.astro';
+
+<ReadMore>Aprende sobre el uso de [componentes de framework de UI](/es/core-concepts/framework-components/) en tu proyecto Astro.</ReadMore>
```
Comment (Waxer59) [inline src/content/docs/es/core-concepts/astro-components.mdx:383]:
```suggestion
<ReadMore>Aprende sobre el uso de [componentes de frameworks UI](/es/core-concepts/framework-components/) en tu proyecto Astro.</ReadMore>
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠! te dejo una sugerencia

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Pull request merged**